usaspending_api/api_contracts/contracts/v2/agency/toptier_code/awards/new/count.md

```diff
@@ -0,0 +1,76 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Overview of new awards for Agency [/api/v2/agency/{toptier_code}/awards/new/count{?fiscal_year,agency_type,award_type_codes}]
+
+Return the count of new Awards under the Agency
+
+## GET
+
++ Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "number"
+            }
+    + Parameters
+        + `toptier_code`: 086 (required, number)
+            The toptier code of an agency (could be a CGAC or FREC) so only numeric character strings of length 3-4 are accepted.
+        + `fiscal_year` (optional, number)
+            The desired appropriations fiscal year. Defaults to the current FY
+        + `agency_type` (optional, enum[string])
+            The agency type to pull the overview for.
+            + Default: `awarding`
+            + Members
+                + `awarding`
+                + `funding`
+        + `award_type_codes` (optional, array[AwardTypeCodes], fixed-type)
+            + Default: `["02", "03", "04", "05", "07", "08", "10", "06", "09", "11", "A", "B", "C", "D", "IDV_A", "IDV_B", "IDV_B_A", "IDV_B_B", "IDV_B_C", "IDV_C", "IDV_D", "IDV_E"]`
```

Comment:
This default example seems a little tricky. The API will not accept quotes or spaces in this list

Reply:
You also don't need to provide all the type codes as a default - if you provide a blank/null for the award type code filter, the filter won't get added to the elasticsearch query and it'll work as if you are filtering by all type codes

Reply:
Fine with removing it for me. Dredd keeps complaining about the default anyways.